src/writer.jl

```diff
@@ -355,10 +355,8 @@ end
 function intelligent_language(lang::String)
     if lang == "documenter-ansi"
         "ansi"
-    elseif lang ∈ ("ansi", "julia-repl", "@doctest", "@repl")
-        "julia /julia>/"
-    elseif lang ∈ ("@example",)
-        "julia"
+    elseif lang ∈ ("ansi", "julia-repl", "@repl", "@example", "@doctest")
+        "julia" # /julia>/ ? why is this one needed it, where?
```

Comment:
`julia-repl` and `doctest` are probably the two places we need it.  The syntax basically indicates to Shiki.js that it should highlight the `julia>` string wherever it sees it.  Unfortunately it's not working...we can remove it for now and re-instate it at some later point.

```suggestion
        "julia" # /julia>/ TODO: implement this highlighting for the `julia-repl` and `@doctest` languages.